### PR TITLE
fix: add SpokePool2.5 contracts

### PR DESCRIPTION
**Pull request by dohaki**
I noticed that this branch was not merged to `stage` and only to `master`. After merging this one, we should sync both branches again

### CODE REVIEW

File: src/modules/scraper/utils.ts [hunk 1]
```diff
@@ -24,8 +24,17 @@ export function splitBlockRanges(
           address: contracts[i].address,
           acrossVersion: contracts[i].acrossVersion,
         });
-      } else {
-        continue;
+      } else if (
+        contracts[i].startBlockNumber <= from &&
+        contracts[i + 1].startBlockNumber <= from &&
+        i === contracts.length - 2
```
Comment (james-a-morris) [inline src/modules/scraper/utils.ts:30]: Should we place this earlier in the `else if` condition?

**Review by james-a-morris — APPROVED** [review]
LGTM - left a nit

**Review by amateima — APPROVED** [review]
We can also reset stage to master and force push, right? afaik there's no work on the staging branch that needs to be pushed to master

**Comment by dohaki** [comment]
> We can also reset stage to master and force push, right? afaik there's no work on the staging branch that needs to be pushed to master

@amateima Yes, I think so too. I will close this PR then and reset `stage` to `master`.

**Pull request closed without merging**